packages/wonder-blocks-modal/src/components/__docs__/modal-header.argtypes.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import * as React from "react";
+import {
+    Breadcrumbs,
+    BreadcrumbsItem,
+} from "@khanacademy/wonder-blocks-breadcrumbs";
+import Link from "@khanacademy/wonder-blocks-link";
+
+const BreadcrumbsMappings: {[key: string]: React.Node} = {
+    none: null,
+    "lesson path": (
```

Comment:
**praise:** Ohh that's a nice touch! I don't know why I didn't use strings with the Popover mappings :).